pages/common/az-upgrade.md

```diff
@@ -0,0 +1,13 @@
+# az upgrade
+
+> Upgrade Azure CLI and Extensions.
+> Part of `az`, the command-line client for Microsoft Azure.
+> More information: <https://learn.microsoft.com/cli/azure/reference-index?view=azure-cli-latest#az-upgrade>.
+
+- Upgrade Azure CLI and Extensions:
+
+`az upgrade --all {{true|false}}`
```

Comment:
if `false` is selected, what isn't updated?

Reply:
The --all flag enables Extension upgrades. So if false is selected, it won't upgrade the Extensions.

Reply:
We should add a different command example for just updating Azure CLI.

Reply:
I should probably remove the ``{{true|false}}`` as it can be confusing. It's optional anyways and defaults to ``true``.

Reply:
Good idea.